### ISSUE TITLE
Compare time cost between mobilenetV1 and shufflenetV1、V2

### ISSUE THREAD

**Issue by Wallace00V** [megvii-model/ShuffleNet-Series#25]
think you for the nice jobs of ShuffleNet series.
i test time cost while finetuning mobilenet and shufflenet in my task in the below condition:
One GPU V100
Batchsize=8
Input size of W,H=64,128
Two labels classification
 
My time cost code is 
```
    start = time.time()
    output = model(img)
    torch.cuda.synchronize()
    end = time.time()
```
 
Here is the time cost:
Mobilenetv1_0.25:   ~0.0035s
MobilenetV1_0.5:    ~0.0036s
Shufflenetv1_0.5(g=3):     ~0.0095s
Shufflenetv2_0.5:      ~0.009s

According to the paper of Shufflenetv2，their GPU speed should be :  Mobilenetv1_0.25 > Shufflenetv2_0.5 > MobilenetV1_0.5 > Shufflenetv1_0.5 , so what is the problem?

**Comment by nmaac** [comment]
Hi @Wallace00V ,

The GPU usage rate should be at least 80% to get accurate speed test result. If you are using V100, you should increase the batch size to ensure the GPU usage rate.

**Comment by luoduo21** [comment]
Hello, have you tried shufflev2 +? On the GPU, the speed of shufflev2 and shufflev2 + is almost the same as when training, but when test, batchsize = 1, shufflev2 + is only half the speed of shufflev2, which is also slower than mobilenetv3. But the speed is normal on the cpu .

**Comment by nmaac** [comment]
Hi @luoduo21 ,

The speed measurements on CPU and GPU are quite different.  Because of GPU's parallelism characteristics, **the GPU usage percentage should be the same** to get the accurate speed comparison. ShuffleNetV2 uses lower GPU rate, so using a batch size of 1 usually gets an inaccurate measurement.
